### PR TITLE
added eslint question

### PR DESCRIPTION
**Pull request by WLefferts**
When the user does not want to use TypeScript they will get the question if they want to use ESLint. If they choose 'Y', a new JSON file will be written and extra devdependencies will be installed.

### CODE REVIEW

**Comment by WLefferts** [comment]
I am not completely certain if the dev dependencies get installed correctly, but I think they do.

File: lib/App.js [hunk 1]
```diff
@@ -2460,6 +2472,9 @@ class App {
     await writeFileAsync(path.join(appPath, mainFile), '');
     await writeFileAsync(path.join(appPath, 'README.md'), `# ${appJson.name.en}\n\n${appJson.description.en}`);
     await writeFileAsync(path.join(appPath, 'README.txt'), `${appJson.description.en}\n`);
+    if (answers.eslint) {
+      await writeFileAsync(path.join(appPath, 'eslintrc.json'), '{ "extends": "athom" }');
```
Comment (WeeJeWel) [inline lib/App.js:2476]: Use 'JSON.stringify({ ... }, false, 2'

Comment (WeeJeWel) [inline lib/App.js:2476]: '.eslintrc.json'

**Review by WeeJeWel — CHANGES_REQUESTED** [review]

**Review by tjallingt — COMMENTED** [review]
Seems pretty good, but theres two eslint errors :)

**Review by tjallingt — APPROVED** [review]

**Review by WeeJeWel — APPROVED** [review]

**Pull request merged**